tests/test_pipeline_script.py

```diff
@@ -86,13 +87,6 @@ def test_run_script_dag(virtualenv, tmp_path):
         ]
     )
 
-    virtualenv.run(
-        "pip install pandas scikit-learn", capture=False, cd=str(tmp_path)
-    )
+    os.chdir(str(tmp_path))
 
-    virtualenv.run(
-        "python sliced_housing_multiple_script_dag.py",
-        capture=False,
-        # Run in current root lineapy so that relative paths are accurate
-        cd=str(tmp_path),
-    )
+    subprocess.check_call(["python", "sliced_housing_multiple_script_dag.py"])
```

Comment:
getting rid of virtual env speeds up the tests from 100s to about 2 seconds. (Needed some python file change to trigger workflows in prev versions. not removing it since its a positive change)

Reply:
Just to make sure, are we intentionally retaining `virtualenv` input arg in the function?